js/web/lib/wasm/jsep/backend-webgpu.ts

```diff
@@ -97,6 +97,14 @@ export class WebGpuBackend {
 
   env: Env;
 
+  profilingMode(): 'off'|'default'|undefined {
+    return this.env.webgpu.profilingMode;
+  }
+
+  validateInputContent(): 'off'|'default'|undefined {
+    return this.env.webgpu.validateInputContent;
+  }
+
```

Comment:
is there a reason for adding these functions? we can access 'env' object in our code.

Reply:
Removed.